tests/app/main/test_frameworks.py

```diff
@@ -3395,94 +3392,53 @@ def _send_email(self, clarification_question):
+        """
+
         if is_called:
-            assert mandrill_send_email.call_count == 1
+            assert notify_send_email.call_count == times_called
         else:
-            assert mandrill_send_email.call_count == 0
+            assert notify_send_email.call_count == 0
 
         if is_called:
-            mandrill_send_email.assert_any_call(
+            notify_send_email.assert_any_call(
                 mock.ANY,  # self
-                to_email_addresses="digitalmarketplace@mailinator.com",
-                from_email_address="enquiries@digitalmarketplace.service.gov.uk",
-                from_name="Test Framework Supplier",
-                subject="Test Framework clarification question",
-                email_body=FakeMail('Supplier ID:'),
-                tags=["clarification-question"],
-                reply_to="enquiries@digitalmarketplace.service.gov.uk",
+                to_email_address="digitalmarketplace@mailinator.com",
+                template_name_or_id='8574484f-9907-44c0-b9d5-1120beb64ef0',
+                personalisation={"framework_name": "Test Framework", "supplier_id": 1234,
+                                 "clarification_question": clarification_question}
             )
 
         if succeeds:
             notify_send_email.assert_any_call(
                 mock.ANY,  # DMNotifyClient
-                "email@email.com",
-                template_name_or_id=mock.ANY,
+                to_email_address="email@email.com",
+                template_name_or_id='1a8a3408-49ef-486f-a6c9-8557d1a0dc63',
                 personalisation={'user_name': 'Năme', 'framework_name': 'Test Framework',
                                  'clarification_question_text': clarification_question},
-                reference=mock.ANY,
-                reply_to_address_id=mock.ANY
+                reference='clarification-question-confirm-8yc90Y2VvBnVHT5jVuSmeebxOCRJcnKicOe7VAsKu50=',
+                reply_to_address_id='24908180-b64e-513d-ab48-fdca677cec52'
             )
 
-    def _assert_application_email(self, mandrill_send_email, *email_body_content, succeeds=True):
-
-        if succeeds:
-            assert mandrill_send_email.call_count == 1
-        else:
-            assert mandrill_send_email.call_count == 0
-
-        if succeeds:
-            mandrill_send_email.assert_called_with(
-                mock.ANY,  # self
-                to_email_addresses="digitalmarketplace@mailinator.com",
-                from_email_address="enquiries@digitalmarketplace.service.gov.uk",
-                from_name="Test Framework Supplier",
-                subject="Test Framework application question",
-                email_body=FakeMail(*email_body_content),
-                tags=["application-question"],
-                reply_to="email@email.com",
-            )
-
-    def test_should_call_send_email_with_correct_params(self, mandrill_send_email, notify_send_email, s3):
+    @mock.patch('dmutils.s3.S3')
```

Comment:
Where are we using S3 in this view? Do we need to patch it?

Reply:
Yep. This test calls `_send_email` on l.3387, which posts to https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/841d428158e2508c30d7a2081b87aca9c35b2073/app/main/views/frameworks.py#L685, which instantiates an s3 object. I could move the patch up to `_send_email`, where it's closer to what's using it?

Reply:
Yeah, that's much neater. I'll do that

Reply:
I think it's the GET view that uses the S3 object, not the POST one?

Reply:
🤔 yeah, you're right. So why does it break so hard when I take it out...

Reply:
Ah, the POST view returns a redirect to the success page. So we do need the patch after all (my mistake!)